### ISSUE TITLE
Expose option to turn off LimitSchemaGraphToTopLevelEntity

### ISSUE THREAD

**Issue by wbreza** [rbeauchamp/Swashbuckle.OData#58]
When the swagger definitions of my API are being generated I'm missing sub objects within the graph.  I've tracked it down to the `LimitSchemaGraphToTopLevelEntity` `IDocumentFilter` being added by default.

Can you expose a configuration property to enable/disable this?

**Comment by bigred8982** [comment]
@rbeauchamp would the easiest solution be to just remove the code that [automatically adds the document filter](https://github.com/rbeauchamp/Swashbuckle.OData/blob/master/Swashbuckle.OData/DefaultCompositionRoot.cs#L120)?

The ```LimitSchemaGraphToTopLevelEntity``` could be changed to public and then added if needed.

```C#
GlobalConfiguration.Configuration.EnableSwagger(c =>
{
    c.DocumentFilter<LimitSchemaGraphToTopLevelEntity>();
    c.CustomProvider(defaultProvider => new ODataSwaggerProvider(defaultProvider, c));
})
.EnableSwaggerUi(c => {});
```

**Comment by rbeauchamp** [comment]
@wbreza @nealeduncan @bigred8982 This feature is now available in v2.16.0 and described in the [documentation](https://github.com/rbeauchamp/Swashbuckle.OData#include-navigation-properties-in-your-entity-swagger-models). Please give it a try and let me know if you find any issues. Thanks!
